docs/operating-modes.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Operating modes
+sidebar_position: 25
+custom_edit_url: https://github.com/stryker-mutator/stryker-net/edit/master/docs/operating-modes.md
+---
+# Operating modes
+
+There are currently two different ways to run Stryker:
+
+- From the test project context
+- From the target project context
```

Comment:
```suggestion
- From the source project context
```

@richardwerkman what do we want to call this? We've called it target project for a long time but that really only makes sense from the context of the test project. It doesn't really make sense anymore now that we can run stryker from different contexts.

Reply:
Source makes more sense now for this. And should we call to solution one "source solution context"? Once it's done of course